portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,66 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    grid-gap: 15px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.header {
+    padding-top: 10px;
+    float: right;
 }
+
+li {
+    list-style-type: none;
+    text-transform: uppercase;
+    font-size: 18px;
+    margin-right: 25px;
+    display: inline;
+    font-weight: 700px;
+}
+
+h4 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-size: 18px;
+    font-weight: 700px;
+    text-transform: uppercase;
+    color: #151426;
+}
+
+img {
+    width: 100%;
+    height: auto;
+}
+
+.quote {
+    grid-column: 1/3;
+}
+
+.about-me {
+    grid-column: 1/3;
+}
+
+.experience {
+    grid-column: 1/3;
+}
+
+h1 {
+    font-family: 'Playfair Display';
+    font-size: 50px;
+    padding-top: 100px;
+}
+
+p {
```

Comment:
I would give these a better naming than just `p` it would be easier when you are referring back to your code